include/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# FreeRTOS internal cmake file. Do not use it in user top-level project
+
+add_library(freertos_kernel_include INTERFACE)
+
+target_include_directories(freertos_kernel_include
+    INTERFACE
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
```

Comment:
I'm not familiar with the usage of INSTALL_INTERFACE vs BUILD_INTERFACE. Is this necessary?

Reply:
If someone runs a CMake install script, the INSTALL_INTERFACE generator expression is used instead of the BUILD_INTERFACE generator.

I think the distinction is only needed when specifying absolute paths for INTERFACE include directories. So, I've simplified both to use a relative path.